spark/v3.1/spark/src/test/java/org/apache/iceberg/spark/actions/TestCreateActions.java

```diff
@@ -517,7 +518,9 @@ public void testSparkTableReservedProperties() throws Exception {
     String source = sourceName("test_reserved_properties_table");
     String dest = destName(destTableName);
     createSourceTable(CREATE_PARQUET, source);
-    assertSnapshotFileCount(SparkActions.get().snapshotTable(source).as(dest), source, dest);
+    SnapshotTable action = SparkActions.get().snapshotTable(source).as(dest);
+    action.tableProperty(TableProperties.FORMAT_VERSION, "1");
```

Comment:
Does this test need to be with Version1, or is this just to minimize changes

Reply:
This actually has to be v1 as the test verifies it is v1 and then does an update to v2.

Reply:
Makes sense, didn't get that from the test name